src/prometheus_alert_rules/host_health.rules

```diff
@@ -0,0 +1,14 @@
+groups:
+- name: HostHealth
+  rules:
+  - alert: HostAvailable
+    expr: up < 1 or absent(up)
```

Comment:
There are some subtle difference we were discussing which I wonder if would give any advantage by separating into different alerts:
- "HostOffline", when `absent(up)`. Would fire when remote write hasn't sent anything for a while.
- "HostDown", when `up < 1`. Would fire when remote write succeeds but scrape fails.